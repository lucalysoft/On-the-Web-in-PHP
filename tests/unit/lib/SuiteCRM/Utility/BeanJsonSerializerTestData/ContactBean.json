{
  "id": "349e0ec4-8c38-b3a1-8dba-5b1fa8875c12",
  "date_entered": "2018-06-12 11:01:34",
  "date_modified": "2018-06-12 11:01:34",
  "modified_user_id": "1",
  "created_by": "1",
  "description": null,
  "deleted": "0",
  "assigned_user_id": "seed_max_id",
  "salutation": "Ms.",
  "first_name": "Cindy",
  "last_name": "Espana",
  "title": "Director Operations",
  "photo": null,
  "department": "Genetics",
  "do_not_call": "0",
  "phone_home": "(445) 163-3872",
  "phone_mobile": "(978) 836-3300",
  "phone_work": "(711) 112-3512",
  "phone_other": null,
  "phone_fax": null,
  "lawful_basis": null,
  "date_reviewed": null,
  "lawful_basis_source": null,
  "primary_address_street": "345 Sugar Blvd.",
  "primary_address_city": "Salt Lake City",
  "primary_address_state": "CA",
  "primary_address_postalcode": "58821",
  "primary_address_country": "USA",
  "alt_address_street": null,
  "alt_address_city": null,
  "alt_address_state": null,
  "alt_address_postalcode": null,
  "alt_address_country": null,
  "assistant": null,
  "assistant_phone": null,
  "lead_source": "Web Site",
  "reports_to_id": null,
  "birthdate": null,
  "campaign_id": null,
  "joomla_account_id": null,
  "portal_account_disabled": null,
  "portal_user_type": "Single",
  "age": null,
  "id_c": null,
  "jjwg_maps_lng_c": null,
  "jjwg_maps_lat_c": null,
  "jjwg_maps_geocode_status_c": null,
  "jjwg_maps_address_c": null,
  "email1": "kid79@example.co.jp",
  "modified_by_name": "Administrator",
  "created_by_name": "Administrator",
  "assigned_user_name": "Max Jensen",
  "account_name": "Start Over Trust",
  "account_id": "39363e96-5eed-0221-9889-5b1fa86ebb52",
  "opportunity_role_fields": "",
  "report_to_name": "",
  "campaign_name": "",
  "c_accept_status_fields": "",
  "m_accept_status_fields": "",
  "e_invite_status_fields": "",
  "e_accept_status_fields": ""
}