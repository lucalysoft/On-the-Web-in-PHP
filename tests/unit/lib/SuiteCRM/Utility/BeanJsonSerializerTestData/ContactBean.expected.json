{
  "id": "00000000-0000-0000-0000-000000000000",
  "meta": {
    "created": {
      "date": "2018-06-12 11:01:34",
      "user_id": 1,
      "user_name": "Administrator"
    },
    "modified": {
      "date": "2018-06-12 11:01:34",
      "user_id": 1,
      "user_name": "Administrator"
    },
    "assigned": {
      "user_id": "seed_max_id",
      "user_name": "Max Jensen"
    }
  },
  "name": {
    "salutation": "Ms.",
    "first": "Cindy",
    "last": "Espana"
  },
  "account": {
    "title": "Director Operations",
    "department": "Genetics",
    "id": "39363e96-5eed-0221-9889-5b1fa86ebb52",
    "name": "Start Over Trust"
  },
  "phone": {
    "home": 4451633872,
    "mobile": 9788363300,
    "work": 7111123512,
    "other": null,
    "fax": null
  },
  "address": {
    "primary": {
      "street": "345 Sugar Blvd.",
      "city": "Salt Lake City",
      "state": "CA",
      "postalcode": 58821,
      "country": "USA"
    },
    "alt": {
      "street": null,
      "city": null,
      "state": null,
      "postalcode": null,
      "country": null
    }
  },
  "assistant": {
    "name": null,
    "phone": null
  },
  "reports_to": {
    "id": null,
    "name": ""
  },
  "campaign": {
    "name": "",
    "id": null
  },
  "lead": {
    "type": "Web Site"
  },
  "birthdate": null,
  "age": null,
  "email": [
    "kid79@example.co.jp"
  ],
  "description": ""
}